### ISSUE TITLE
resnet_prep() Transforming y_train and y_valid in call to model.fit()

### ISSUE THREAD

**Issue by Andrewishere502** [Andrewishere502/NeoCounterAI#1]
resnet_prep() should only be used on the X (image) data, but was used on the y (labels) too.

`
history = model.fit(
    resnet_prep(X_train),
   # y_train should not be transformed by resnet_prep
    resnet_prep(y_train), 
    # y_valid should not be transformed by resnet_prep
    validation_data=(resnet_prep(X_valid), resnet_prep(y_valid)),
    class_weight=class_weight,
    epochs=25,
    callbacks=[csv_logger, early_stop]
    )
`
